integration/hive/hive-openlineage-hook/integrations/container/simpleCtasComplete.json

```diff
@@ -0,0 +1,152 @@
+  "job": {
+    "namespace": "default",
+    "name": "createtable_as_select.test.result_t"
+  },
+  "inputs": [
+    {
+      "namespace": "hive://localhost:9083",
+      "name": "test.employees",
+      "facets": {
+        "schema": {
+          "fields": [
+            {
+              "name": "id",
+              "type": "int"
+            },
+            {
+              "name": "name",
+              "type": "string"
+            },
+            {
+              "name": "team",
+              "type": "int"
+            }
+          ]
+        },
+        "symlinks": {
+          "identifiers": [
+            {
+              "namespace": "file",
+              "name": "/opt/hive/data/warehouse/test.db/employees",
+              "type": "TABLE"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "namespace": "hive://localhost:9083",
+      "name": "test.managers",
+      "facets": {
+        "schema": {
+          "fields": [
+            {
+              "name": "id",
+              "type": "int"
+            },
+            {
+              "name": "name",
+              "type": "string"
+            },
+            {
+              "name": "team",
+              "type": "int"
+            }
+          ]
+        },
+        "symlinks": {
+          "identifiers": [
+            {
+              "namespace": "file",
+              "name": "/opt/hive/data/warehouse/test.db/managers",
+              "type": "TABLE"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "namespace": "hive://localhost:9083",
+      "name": "test.teams",
+      "facets": {
+        "schema": {
+          "fields": [
+            {
+              "name": "id",
+              "type": "int"
+            },
+            {
+              "name": "type",
+              "type": "int"
+            },
+            {
+              "name": "building",
+              "type": "string"
+            }
+          ]
+        },
+        "symlinks": {
+          "identifiers": [
+            {
+              "namespace": "file",
+              "name": "/opt/hive/data/warehouse/test.db/teams",
+              "type": "TABLE"
+            }
+          ]
+        }
+      }
+    }
+  ],
+  "outputs": [
+    {
+      "namespace": "hive://localhost:9083",
+      "name": "test.result_t",
+      "facets": {
+        "schema": {
+          "fields": [
+            {
+              "name": "type",
+              "type": "int"
+            },
+            {
+              "name": "building",
+              "type": "string"
+            },
+            {
+              "name": "manager",
+              "type": "string"
+            },
+            {
+              "name": "employee",
+              "type": "string"
+            }
+          ]
+        },
+        "symlinks": {
+          "identifiers": [
+            {
+              "namespace": "file",
+              "name": "/opt/hive/data/warehouse/test.db/result_t",
+              "type": "TABLE"
```

Comment:
Maybe this should be `LOCATION`, as dataset describes table

Reply:
I think that it's defined the same way when spark integration handles hive tables

Reply:
besides I don't think I've seen any other type of symlink being produced by any integration, so not really sure what is appropriate

Reply:
In Spark datasets looks like `file:/some/path` and symlink is `TABLE` + `hive://host:port`. Here it is vice versa

Reply:
yeah, makes sense 

@mobuchowski @pawel-big-lebowski  what do you think about the `LOCATION` type? If there is a consensus then I can set it to whatever

Reply:
just noticed that this would require adding this in the client code and in this PR I would like to keep changes only to the integration, this can be added as separate PR though